### PR TITLE
Create sensor.zestimate.markdown

### PR DESCRIPTION
**Pull request by jcconnell**
Add documentation for the Zestimate sensor.

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.zestimate.markdown [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+layout: page
+title: "Zestimate"
+description: "Instructions on how to integrate the Zestimate sensor into Home Assistant."
+date: 2018-03-02 3:10
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Sensor
+ha_release: 0.64
```
Comment (MartinHjelmare) [inline source/_components/sensor.zestimate.markdown]: 0.65

File: source/_components/sensor.zestimate.markdown [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  - platform: zestimate
+    name: <your name>
+    api_key: <your API key>
+    zpid:
+      - <your zpid 1>
+      - <your zpid 2>
+      - <your zpid ...>
+```
+
+Configuration options for the Crime Reports Sensor:
```
Comment (MartinHjelmare) [inline source/_components/sensor.zestimate.markdown]: Stale comment.

Comment (MartinHjelmare) [reply source/_components/sensor.zestimate.markdown]: Please instead use the configuration tag:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/sensor.zestimate.markdown [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+- **api_key** (*Required*): Your Zillow API key.
+- **zpid** (*Required*): List of Zillow property IDs.
+- **name** (*Optional*): Name your sensor.
+
+
+## Notes
```
Comment (MartinHjelmare) [inline source/_components/sensor.zestimate.markdown]: No notes yet.

**Comment by jcconnell** [comment]
@MartinHjelmare, I think I addressed all your changes. I followed some of the additional advice available at the configuration [docs](https://home-assistant.io/developers/documentation/create_page/#configuration) by removing the non-required name variable. I'm still not sure if I've done it correctly. Possible to see an example of a doc that uses it the correct way if not?

File: source/_components/sensor.zestimate.markdown [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+```yaml
+sensor:
+  - platform: zestimate
+    api_key: <your API key>
+    zpid:
+      - <your zpid 1>
+      - <your zpid 2>
+```
+
+Configuration options for the Zestimate Sensor:
```
Comment (MartinHjelmare) [inline source/_components/sensor.zestimate.markdown]: The configuration tag will create a header on its own, so I don't think you need this header here.

Comment (jcconnell) [reply source/_components/sensor.zestimate.markdown]: Should the block starting on line 31 still have the code highlighting? Should I leave the example starting on line 22 as well?

File: source/_components/sensor.zestimate.markdown [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+```yaml
+sensor:
+  - platform: zestimate
+    api_key: <your API key>
+    zpid:
+      - <your zpid 1>
+      - <your zpid 2>
+```
+
+```yaml
```
Comment (MartinHjelmare) [inline source/_components/sensor.zestimate.markdown]: Sorry, I missed this, and thanks for pointing this out! Please remove the code block around here configuration tags.

**Comment by jcconnell** [comment]
Thanks for everyone's help! Still learning but I'm really enjoying contributing.

**Pull request merged**